<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineListViewer</class>
 <widget class="QWidget" name="MachineListViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MachineListViewer</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widgetMenu" native="true">
     <layout class="QGridLayout" name="gridLayoutWidgetMenu" columnstretch="2,0,1,0,0,0,0,0,0,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item row="0" column="9">
       <widget class="QToolButton" name="toolButtonUpdateView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Update view</string>
        </property>
        <property name="statusTip">
         <string>Update view</string>
        </property>
        <property name="text">
         <string>Update</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/reload.png</normaloff>:/data/img/reload.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QToolButton" name="toolButtonConfigureFilters">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Configure view filters</string>
        </property>
        <property name="statusTip">
         <string>Configure view filters</string>
        </property>
        <property name="text">
         <string>Filters</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/filter.png</normaloff>:/data/img/filter.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QToolButton" name="toolButtonViewModeToggle">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Toggle flat / tree view mode</string>
        </property>
        <property name="statusTip">
         <string>Toggle flat / tree view mode</string>
        </property>
        <property name="text">
         <string>Flat</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/view_detail.png</normaloff>:/data/img/view_detail.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLCDNumber" name="lcdNumberRecordCount">
        <property name="toolTip">
         <string>Number of records in view</string>
        </property>
        <property name="statusTip">
         <string>Number of records in view</string>
        </property>
        <property name="digitCount">
         <number>6</number>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolButtonToolsMenu">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Tools menu</string>
        </property>
        <property name="statusTip">
         <string>Tools menu</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/toolbox.png</normaloff>:/data/img/toolbox.png</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="8">
       <widget class="Line" name="line2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxViewName">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Enter a unique name for this view or select an existing one</string>
        </property>
        <property name="statusTip">
         <string>Enter a unique name for this view or select an existing one</string>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QToolButton" name="toolButtonVisibleColumns">
        <property name="toolTip">
         <string>Setup visible columns</string>
        </property>
        <property name="statusTip">
         <string>Setup visible columns</string>
        </property>
        <property name="text">
         <string>Columns</string>
        </property>
        <property name="icon">
         <iconset resource="qmc2.qrc">
          <normaloff>:/data/img/columns.png</normaloff>:/data/img/columns.png</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="Line" name="line1">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButtonToggleMenu">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>6</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>6</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Toggle menu</string>
     </property>
     <property name="statusTip">
      <string>Toggle menu</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QTreeView" name="treeView">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideMiddle</enum>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="allColumnsShowFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxViewName</tabstop>
  <tabstop>toolButtonToolsMenu</tabstop>
  <tabstop>toolButtonViewModeToggle</tabstop>
  <tabstop>toolButtonConfigureFilters</tabstop>
  <tabstop>toolButtonVisibleColumns</tabstop>
  <tabstop>toolButtonUpdateView</tabstop>
  <tabstop>toolButtonToggleMenu</tabstop>
  <tabstop>treeView</tabstop>
 </tabstops>
 <resources>
  <include location="qmc2.qrc"/>
 </resources>
 <connections/>
</ui>
